provisioning/service/tests/Config/TwinStateTests.cs

```diff
@@ -75,15 +75,12 @@ public void TwinStateSucceedOnNull()
             var initialTwin = new InitialTwinState
             {
                 Tags = tags,
-                Properties =
```

Comment:
Basically fixing this nonsense. It is correct for serialization, but creates an undue burden on the user.